docs/lvn-integration.md

```diff
@@ -0,0 +1,42 @@
+# Guide: How to Obtain a Vonage LVN (Long Virtual Number)
+
+This guide provides step-by-step instructions on how to obtain a Vonage LVN (Long Virtual Number) and link it to your application. A Vonage LVN allows you to receive calls, SMS, and MMS messages through webhooks in your application.
+
+## Table of Contents
+1. [Buy a Vonage LVN](#1-buy-a-vonage-lvn)
+2. [Link the Number to Your Application](#2-link-the-number-to-your-application)
+3. [Conclusion](#conclusion)
+
+## 1. Buy a Vonage LVN
+
+To obtain a Vonage LVN, follow these steps:
+
+1. Go to the [Vonage Dashboard](https://dashboard.nexmo.com/) and navigate to **Numbers** in the side navigation menu.
+2. Click on **Buy Numbers**.
+3. Use the available filters to search for a specific type of number:
+   - Select the desired **Region** (e.g., United Kingdom).
+   - Choose the desired **Feature** (e.g., SMS, MMS, and Voice).
```

Comment:
maybe you can mention to clear that only USA lvns supports MMS

Reply:
I'm not so sure that's strictly true, as some numbers need to be setup by Vonage sales and support.

Reply:
There's a generic note saying that some features might be supported only in specific regions